mobile/README.md

```diff
@@ -114,11 +159,90 @@ to your project to support the generated code.
 
 ### Android
 
+####First option:
+
 In order to generate protobuf definitions for Android, add `--java_out=.`
 
 to the first `protoc` invocation found in
 [ `gen_protos.sh` ](../lnrpc/gen_protos.sh). Then, run `make rpc`.
 
+
+####Second option (preferable):
+
+- You have to install the profobuf plugin to your Android application. 
+Please, follow this link https://github.com/google/protobuf-gradle-plugin.
+- Add this line to your `app build.gradle` file.
+```shell
+classpath "com.google.protobuf:protobuf-gradle-plugin:0.8.17"
+```
+- Create a `proto` folder under the `main` folder.
+
+![proto_folder](images/proto_folder.png)
+
+- Add `aar` file to libs folder.
+
+- After that add these lines to your `module's` `build.gradle` file:
+
+```shell
+plugins {
+    id "com.google.protobuf"
+}
+
+android {
+    sourceSets {
+        main {
+            proto {
+
+            }
+        }
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: "libs", include: ["*.jar"])
+    implementation "com.google.protobuf:protobuf-javalite:${rootProject.ext.javalite_version}"
+}
+
+protobuf {
+    protoc {
+        artifact = "com.google.protobuf:protoc:${rootProject.ext.protoc_version}"
+    }
+    generateProtoTasks {
+        all().each { task ->
+            task.builtins {
+                java {
+                    option "lite"
+                }
+            }
+        }
+    }
+}
+```
+- Then, copy all the proto files from `lnd/lnrpc` to your `proto` folder, saving the structure.
+- Build the project and in `build` folder you have to see the generated Java proto files.
+
+####Note:
+
+If Android Studio tells you that the `aar` file cannot be included into the `app-bundle`, this is a workaround:
+
+1. Create a separate gradle module
+2. Remove everything from there and leave only `aar` and `build.gradle`.
+
+![separate_gradle_module](images/separate_gradle_module.png)
```

Comment:
nit: add the images to a `mobile/docs` sub folder instead to indicate that they aren't needed for the build itself?

Reply:
These are just screenshots, I don't think that Android users would think that those screenshots are a part of build) But okay, if it's clearer, I will do it.